### PR TITLE
add tests for location OR stanford variants

### PR DESCRIPTION
**Pull request by ndushay**
and add tests for variants of multiple files getting assigned rights

plus a bit of refactoring:
- to tighten up some of the descriptive wording.
- to shorten rights_auth_spec file which was getting really long.

connects to #18 ... which really didn't need a FIX ... just a clarification of the actual requirements and changes to the test data ...

### CODE REVIEW

File: spec/rights_auth_location_spec.rb [hunk 1]
```diff
@@ -0,0 +1,233 @@
+require 'spec_helper'
+
+describe Dor::RightsAuth do
+
+  describe 'location based auth' do
+
```
Comment (ndushay) [inline spec/rights_auth_location_spec.rb]: do we care?

File: spec/rights_auth_location_spec.rb [hunk 2]
```diff
@@ -0,0 +1,233 @@
+require 'spec_helper'
+
+describe Dor::RightsAuth do
+
```
Comment (ndushay) [inline spec/rights_auth_location_spec.rb]: do we care?

**Comment by jkeck** [comment]
👍 LGTM

**Comment by ndushay** [comment]
as this is only adding tests, reorganizing a few tests, and removing an inline rubocop comment, I am merging with @jkeck's  LGTM comment.

**Pull request merged**